app/workers/killmail_worker.rb

```diff
@@ -1,29 +1,28 @@
-class KillmailWorker
+class KillmailWorker < ApplicationWorker
   # This Worker will be run when a player is mining something
-
-  include Sidekiq::Worker
-  sidekiq_options retry: false
-
-  def perform(attr, attackers = nil, loot = nil)
+  def perform(body, attackers = nil, loot = nil)
     uri = URI(ENV.fetch("KILLBOARD_URL", "https://killboard.stellar-invictus.com/"))
     req = Net::HTTP::Post.new(uri, 'Content-Type' => 'application/json')
 
-    body = attr
-
     if attackers && (attackers != [])
       temp = []
       attackers.each do |at|
```

Comment:
Can we rename `at` to `attacker` here? Easier to read

Reply:
yep.